src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp

```diff
@@ -1354,7 +1355,7 @@ class npc_tirion_fordring_tft : public CreatureScript
             {
                 // remove glow on ashbringer and tirion
                 me->RemoveAllAuras();
-                SetEquipmentSlots(true);
+                SetEquipmentSlots(false, 13262, EQUIP_NO_CHANGE, EQUIP_NO_CHANGE);
```

Comment:
Should probably be done in DB. Also never use magic numbers

Reply:
> I found Highlord Tirion Fordring (38995) in the database that was equipped with Ashbringer I don't know why I use `SetEquipmentSlots(true); `After the function, using the default equipment will unload the weapon

actually the weapon id referenced in the cpp file is from another ashbringer, one that has a visual effect, you can comment out the "normal" ashbringer id, no effects

I did something that worked great, try to see if it solves the problem :)

I didn't even see that you had already opened a pull. try to take advantage of the pull I opened there
https://github.com/azerothcore/azerothcore-wotlk/pull/15843/commits/7c84ca2a42d6dcb016602628478d134e5d62303b

https://github.com/azerothcore/azerothcore-wotlk/pull/15843/commits/7d4f93f87cf04e5931dea2bcd963b294ad069cc6